docs/tutorials/embedding/OutOfSampleLSE.ipynb

```diff
@@ -0,0 +1,190 @@
+{
```

Comment:
To be honest, it might be better to just have a single tutorial for out-of-sample embedding, called "Out-of-Sample Embedding", which just modifies the ASE one to add the LSE version. I think they're too similar to warrant separate tutorials

Reply:
Just combined them!